.changeset/forty-humans-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@builder.io/qwik': major
+---
+
+crash down when there are two slashes at once
```

Comment:
I don't understand what this means :( Can you phrase it differently? Should we have a test for this issue?